tests/service/SyntaxTreeTests.fs

```diff
@@ -175,17 +175,24 @@ let ParseFile fileName =
             $"%A{ast}\n\n%s{diagnostics}"
         |> normalize
         |> sprintf "%s\n"
+
     let bslPath = $"{fullPath}.bsl"
+    let tmpPath = $"{fullPath}.tmp"
```

Comment:
.tmp => .actual  for immediate clarity about which files is for what?

Reply:
Updated.